src/store-api/src/storage.rs

```diff
@@ -14,35 +14,16 @@
 
 //! Storage APIs.
 
-mod chunk;
 pub mod consts;
 mod descriptors;
-mod engine;
-mod metadata;
-mod region;
 mod requests;
-mod responses;
-mod snapshot;
 mod types;
 
 pub use datatypes::data_type::ConcreteDataType;
 pub use datatypes::schema::{
     ColumnDefaultConstraint, ColumnSchema, Schema, SchemaBuilder, SchemaRef,
 };
 
-pub use self::chunk::{Chunk, ChunkReader};
 pub use self::descriptors::*;
-pub use self::engine::{
-    CloseOptions, CompactionStrategy, CreateOptions, EngineContext, OpenOptions, StorageEngine,
-    TwcsOptions,
-};
-pub use self::metadata::RegionMeta;
-pub use self::region::{
-    CloseContext, CompactContext, FlushContext, FlushReason, Region, RegionStat, WriteContext,
-};
-pub use self::requests::{
-    AddColumn, AlterOperation, AlterRequest, GetRequest, ScanRequest, WriteRequest,
-};
-pub use self::responses::{GetResponse, ScanResponse, WriteResponse};
-pub use self::snapshot::{ReadContext, Snapshot};
-pub use self::types::{SequenceNumber, MIN_OP_TYPE};
+pub use self::requests::ScanRequest;
+pub use self::types::SequenceNumber;
```

Comment:
question(non-blocking): Is there a better place for this little remaining thing?

Reply:
@evenyag cc.

Reply:
We might move ScanRequest to `region.rs` and remove `requests.rs`

Reply:
Seems cannot move to `region.rs` since the `ScanRequest` is also referenced by codes in `store-api`. For now, the `ScanRequest still resides in `requests.rs`.